drivers/adc/ad400x/ad400x.c

```diff
@@ -136,33 +128,32 @@ int32_t ad400x_spi_single_conversion(struct ad400x_dev *dev,
  * @return 0 in case of success, negative error code otherwise.
  */
 int32_t ad400x_init(struct ad400x_dev **device,
-		    struct ad400x_init_param init_param)
+		    struct ad400x_init_param *init_param)
```

Comment:
I think you should split this commit in 2:
 - ad400x Driver update
 - Update project ad400x